src/IdleTracker.cpp

```diff
@@ -69,7 +69,7 @@ LRESULT CALLBACK MouseTracker(int code, WPARAM wParam, LPARAM lParam)
         DWORD curtime = GetTickCount();
         CheckAway(g_dwLastTick, curtime);
 
-        MOUSEHOOKSTRUCTEX *pStruct = (MOUSEHOOKSTRUCTEX *)lParam;
+        MSLLHOOKSTRUCT *pStruct = (MSLLHOOKSTRUCT *)lParam;
```

Comment:
these changes have already gone in, so you'll need to rebase against new master.